porcupine/plugins/filemanager.py

```diff
@@ -25,6 +26,8 @@
 else:
     trash_name = "trash"
 
+copy_path = ""
```

Comment:
It is better to use `None` as the "missing" value. You also need to add an annotation for the type checker to know what else it can be than `None`.

```suggestion
copy_path: Path | None = None
```